operator-metadata/manifests/bundle.clusterserviceversion.yaml

```diff
@@ -790,7 +723,7 @@ spec:
     This release supports only the API version `v1beta2` and CRD version `apiextensions.k8s.io/v1`. 
     If upgrading from AMQ Streams 1.7, migration to `v1beta2` needs to be completed for all AMQ Streams CRDs and CRs before the upgrade to 1.8 is done! 
     If upgrading from AMQ Streams version earlier than 1.7, you need to first install the CRDs from AMQ Streams 1.7 and complete the migration to `v1beta2` for all AMQ Streams CRDs and CRs before the upgrade to 1.8 is done! 
-    For more details about the CRD upgrades, see the [documentation](https://access.redhat.com/documentation/en-us/red_hat_amq/2021.q2/html/deploying_and_upgrading_amq_streams_on_openshift/assembly-upgrade-str#assembly-upgrade-resources-str).
+    For more details about the CRD upgrades, see the [documentation](https://access.redhat.com/documentation/en-us/red_hat_amq/2021.q4/html/deploying_and_upgrading_amq_streams_on_openshift/assembly-upgrade-str#assembly-upgrade-resources-str).
```

Comment:
Did you checked with @PaulRMellor whether it will make it into the Q4 release or only 2022.q1?

Reply:
Waiting to hear @PaulRMellor, invited him as a contributor and sent him a message

Reply:
This kind of thing should be easier now. We're moving away from the quarterly approach (`2021.q4` etc) and to a dedicated Streams splash page that uses the version number (`2.0`).